src/addresses.h

```diff
@@ -93,12 +91,6 @@
 #define RCT2_ADDRESS_CONFIG_RESOLUTION_WIDTH		0x009AB4C2
 #define RCT2_ADDRESS_CONFIG_RESOLUTION_HEIGHT		0x009AB4C4
 
-#define RCT2_ADDRESS_APP_PATH_SLASH					0x009AB4D9
-#define RCT2_ADDRESS_SAVED_GAMES_PATH				0x009AB5DA
-#define RCT2_ADDRESS_SCENARIOS_PATH					0x009AB6E9
-#define RCT2_ADDRESS_LANDSCAPES_PATH				0x009AB7FB
-#define RCT2_ADDRESS_OBJECT_DATA_PATH				0x009AB90E
-#define RCT2_ADDRESS_TRACKS_PATH					0x009ABA1E
```

Comment:
Can you move all of these defines into the depreciated section rather than remove them.